tests/fixtures/folder_fixtures.py

```diff
@@ -22,6 +28,22 @@ def local_folder(tmp_path):
     return local_folder
 
 
+@pytest.fixture
+def local_folder_docker(docker_cluster_pk_ssh_no_auth):
+    args = {
+        "name": "test_folder",
+        "system": docker_cluster_pk_ssh_no_auth,
+        "path": "rh-folder",
+    }
+
+    local_folder_docker = rh.folder(**args)
+    init_args[id(local_folder_docker)] = args
```

Comment:
can you do the same for all the remaining folder fixtures? you should be able to run minimal and have them all test the base resource tests locally.